### ISSUE TITLE
the predicted results are all 0s

### ISSUE THREAD

**Issue by abel-w** [DrSleep/tensorflow-deeplab-lfov#19]
I has been used the data of VOC2012 and the your code to train the models, but the predicts was all 0s during train process. Would you give me some advises. Thank you very much.

**Comment by zwang92** [comment]
[Issue Description]
By using VOCtrainval_11-May-2012 and author's SegmentationClassAug data set to train the network, i also find preds (as following python code) in the SNAPSHOT_DIR are all 0
...
                axes.flat[i * 3 + 2].imshow(decode_labels(preds[i, :, :, 0]))
...

I am using tensorflow r1.4, is there some compability issue here ?
Could anyone else give some suggestions?
thanks

**Comment by cyccty** [comment]
Me too. Data is VOC2012. Tensorflow version is 0.11.0
And the strange phenomenon as following:
First epoch: the predicted results are right, mIoU is reasonable.
Other epochs: the predicted results are all 0, mIoU is very very low.
Same dataset works well by using DeepLab v2.
Could anyone give some solutions?
Thanks.

**Comment by ai1361720220000** [comment]
Could anyone have solved this problem?

**Comment by ShunLu91** [comment]
I used to meet the same problem but I found that it may be the loss that was too large.
